File: CP Lab/Lab2/9.py
# Lab Project by Sarang Dev Saha

# LAB_02 QUE_09
#Read a single digit number from the keyboard and print its corresponding day of week. If number other than 1-7 is entered, print “Invalid Number".

#Taking input fromcthe user
a=int(input("Enter a single digit number:"))
#Using if function to assign a specific day to a number from 1-7
if a==1:
	print("It is Monday")
if a==2:
	print("It is Tuesday")
if a==3:
	print("It is Wednesday")
if a==4:
	print("It is Thursday")
if a==5:
	print("It is Friday")
if a==6:
	print("It is Saturday")
if a==7:
	print("It is Sunday")
else:
	print("Invalid Number")

File: CP Lab/Lab4/9.c
// Lab project by Sarang Dev Saha (20UCS173)
//LAB4_QUES 9
/* Read three numbers from the keyboard. Find the maximum and second maximum
of these numbers. */
# include <stdio.h>
int main() {
    double n1, n2, n3;
    printf("Enter the three numbers: ");
    scanf("%lf %lf %lf", &n1, &n2, &n3);

    // when n1 is the greatest and if n2 or n3 is second largest
    if (n1 >= n2 && n1 >= n3){
        printf("%.2f is the largest number\n", n1);
		if (n2>=n3){
			printf("%2f is the second largest number", n2);
		}
		
		else{
			printf("%2f is the second largest number", n3);
		}
    }

    // if n2 is the greatest and if n1 or n3 is the second largest
    if (n2 >= n1 && n2 >= n3){
        printf("%.2f is the largest number\n", n2);
		if (n1>=n3){
			printf("%2f is the second largest number", n1);
		}
		
		else{
			printf("%2f is the second largest number", n3);
		}
    }

    // if n3 is greatest and if n1 or n2 is the second largest
    if (n3 >= n1 && n3 >= n2){
        printf("%.2f is the largest number\n", n3);
		if (n1>=n2){
			printf("%2f is the second largest number", n1);
		}
		
		else{
			printf("%2f is the second largest number", n2);
		}
    }

    return 0;
}